entrypoint.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
 
 export CODACY_PROJECT_TOKEN=$1
-bash <(curl -Ls https://coverage.codacy.com/get.sh)
+# comma separated list of report files
+report_list=$2
+
+IFS=','
+report_array=$report_list
+params=''
+for report in $report_array
+do
+    if [ ! -z "$report" ]
+    then
+        params="$params -r $report"
+    fi
+done
+
+bash <(curl -Ls https://coverage.codacy.com/get.sh) report $params --partial &&\
```

Comment:
Do you need the `&&\`?

Reply:
Without it, we run the second action even if the first failed. Which shouldn't happen

Reply:
Ah, cool 👍